services/121-service/src/events/events.service.ts

```diff
@@ -259,19 +269,27 @@ export class EventsService {
 
     const events: EventEntity[] = [];
     for (const fieldName of fieldNames) {
+      const key = fieldName as keyof LogEntity;
```

Comment:
I wonder if you can make `keyof LogEntity` the return type of `getRelevantRegistrationViewKeys` so that you can remove this ` as `

Reply:
I have update both methods to make key typed correctly.